# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsGridContainerFrame.h
# Commit: 6b249a6ae76a
# Full Hash: 6b249a6ae76aff5da8ba9ab8311fa596491b58e6
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-05-22 15:28:21
# Regressor Bug: 1465296
# File Overlap Count: 1
# Description:
#   Bug 1465296 part 3 - [css-grid-2] Add a convenience method to get the parent GridContainerFrame for a subgrid.  r=dholbert
# ==============================================================================

diff -r 386b8a935d2b -r 6b249a6ae76a layout/generic/nsGridContainerFrame.h
--- a/layout/generic/nsGridContainerFrame.h	Thu Nov 29 00:18:09 2018 +0100
+++ b/layout/generic/nsGridContainerFrame.h	Thu Nov 29 00:18:09 2018 +0100
@@ -268,6 +268,9 @@
     bool mIsInEdgeTrack;
   };
 
+  /** Return our parent grid container; |this| MUST be a subgrid. */
+  nsGridContainerFrame* ParentGridContainerForSubgrid() const;
+
  protected:
   static const uint32_t kAutoLine;
   // The maximum line number, in the zero-based translated grid.
